pkg/querier/remote_read.go

```diff
@@ -288,10 +284,19 @@ func streamChunkedReadResponses(stream io.Writer, ss storage.ChunkSeriesSet, que
 				return errors.Wrap(err, "write to stream")
 			}
 			chks = chks[:0]
+			frameBytesLeft = initializedFrameBytesLeft(maxBytesInFrame, lbls)
 		}
 		if err := iter.Err(); err != nil {
 			return err
 		}
 	}
 	return ss.Err()
 }
+
+func initializedFrameBytesLeft(maxBytesInFrame int, lbls []mimirpb.LabelAdapter) int {
```

Comment:
[Nit] using remaining instead of left for clarity. 
```suggestion
func initializedFrameBytesRemaining(maxBytesInFrame int, lbls []mimirpb.LabelAdapter) int {
```